CITATION.cff

```diff
@@ -0,0 +1,42 @@
+cff-version: 1.2.0
+title: Metasynth
```

Comment:
Shouldn't we use `MetaSynth` to be consistent?

Reply:
For sure